census/models/utils.js

```diff
@@ -133,13 +133,17 @@ var processEntries = function(data, options) {
     data.submitters = [];
 
     if (options.cascade) { data.entries = cascadeEntries(data.entries, options.year); }
+
+    _.each(data.entries, function(e) {
+      e.computedYCount = e.yCount(data.questions);
```

Comment:
We use computedYCount property for "entries" and "pending" group and we use this property in overview.html in class attribute too  (class="ycount-xxx"  xxx - value of property). But i can not found where this class is using. Seems like we don't need calculate it for rejected.